File: cheating.py
import sys
try:
    r=int(input())
    if r>10:
         raise ValueError
except ValueError:
    sys.exit()
msg=input()
buffer=msg.split()
a=[]
for i in range(0,len(msg)):
    a.append(msg[i])
b=[]
for i in range(0,r):
    c=[]
    for j in range(0,len(a)):
        c.append(0)
    b.append(c)
#print(b)
count=0
for i in range(r-1,0,-1):
    for j in range(0,2*i):
        if j==1 or j==(2*i-1):
            b[i][j]=a[count]
            count+=1
for i in range(0,r):
    for j in range(1,2*i):
        if j==r or j==(2*i-1):
            b[i][j]=a[count]
            count+=1
s=""
for j in range(0,len(a)):
    for i in range(0,r):
        if b[j][i]!=0:
            s=s+str(b[j][i])
print(s)         
        
    

File: cheating.java
import java.util.*;
class cheating
{
    public static void main(String[] args)
    {
        Scanner x=new Scanner(System.in);
        tryS
        {
        int r=x.nextInt();
            if(r>10)
                throw new RunTimeException();
        }
        catch(RunTimeException e)
        {
            System.exit();
        }
        String msg=x.next();
        String line[]=msg.split("");
        int count=0;
        for(int i=0;i<r;i++)
        {
            for(int j=0;j<line.length;j++)
            {
                if(i==0)
                {
                    if(j%8==0)
                    {
                        
                        
                    }
                }
            }
        }
    }
}